src/main/java/ProjectOcean/Model/Course.java

```diff
@@ -173,10 +173,11 @@ public boolean equals(Object o) {
     /**
      * @return a hash code
      */
+    /*
     @Override
     public int hashCode() {
         return Objects.hash(courseCode, courseName, studyPoints, studyPeriod, examiner, examinationMeans, language, requiredCourses, coursePMLink, courseDescription);
-    }
+    } */
```

Comment:
Uncomment?

Reply:
No, this should be deleted. Fixed now. Explanation provided at Tetres comment